src/Features/Context/WebContext.php

```diff
@@ -89,32 +85,32 @@ public function setGoutteDriver()
      *
      * @AfterScenario @remote
      */
-    public function resetGoutteDriver()
+    public function resetGoutteDriver(): void
     {
         $this->minkContext->getMink()->setDefaultSessionName($this->previousMinkSession);
     }
 
-    /**
-     * @return \Surfnet\SamlBundle\Entity\IdentityProvider
-     */
-    public function getIdentityProvider()
+    public function getIdentityProvider(): IdentityProvider
     {
         /** @var RequestStack $stack */
         $stack = $this->kernel->getContainer()->get('request_stack');
         $stack->push(Request::create('https://tiqr.stepup.example.com'));
         $ip = $this->kernel->getContainer()->get('surfnet_saml.hosted.identity_provider');
         $stack->pop();
 
+        if (!$ip instanceof IdentityProvider) {
```

Comment:
I would suggest to rename ip to identityProvder (in all repo's). Especially, because ip means something else in IT

Reply:
I agree with you. We should have used the IdP abbreviation here.